apps/expo-go/android/expoview/src/main/java/host/exp/exponent/exceptions/ManifestException.kt

```diff
@@ -74,15 +70,15 @@ class ManifestException : ExponentException {
                 }
 
                 formattedMessage =
-                  "This project uses SDK $sdkVersionRequired, but this version of Expo Go supports only SDKs ${supportedSdksString("and")}.<br><br>" +
+                  "This project uses SDK $sdkVersionRequired, but this version of Expo Go supports only SDK ${supportedSdk}.<br><br>" +
                   "To open this project:<br>" +
-                  "• Update it to SDK ${supportedSdksString("or")}.<br>" +
+                  "• Update it to SDK ${supportedSdk}.<br>" +
                   "• Install an older version of Expo Go that supports the project's SDK version.<br><br>" +
                   "If you are unsure how to update the project or install a suitable version of Expo Go, refer to the <a href='https://docs.expo.dev/get-started/expo-go/#sdk-versions'>SDK Versions Guide</a>."
               }
-              "SNACK_NOT_FOUND_FOR_SDK_VERSION" -> {
+              "NO_SDK_VERSION_SPECIFIED" -> {
```

Comment:
This fixes a bug accidentally introduced in https://github.com/expo/expo/pull/24310 (duplicate switch case).